IdentityCore/tests/MSIDAccountCacheItemTests.m

```diff
@@ -37,20 +37,28 @@ @implementation MSIDAccountCacheItemTests
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->